phalcon/Http/Request.zep

```diff
@@ -146,6 +146,21 @@ class Request extends AbstractInjectionAware implements RequestInterface
         return this->getBestQuality(this->getLanguages(), "language");
     }
 
+    /**
+     * Gets best language accepted by the browser/client from
+     * _SERVER["HTTP_ACCEPT_LANGUAGE"] and returns the base part of it i.e.
+     * `en` instead of `en-US`
+     */
+    public function getBestLanguageIso() -> string
```

Comment:
I think the method name is a bit confusing. `en-US` is also an iso standard.

Reply:
We shouldn't use Zephir variables in PHP doc. Please consider using `$_SERVER` instead of `_SERVER`. 

```php
/**
 * Gets the preferred ISO locale variant.
 *
 * Gets the preferred locale accepted by the client from the
 * "Accept-Language" request HTTP header and returns the
 * base part of it i.e. `en` instead of `en-US`.
 *
 * Note: This method rely on `$_SERVER["HTTP_ACCEPT_LANGUAGE"]` header.
 *
 * @link https://www.iso.org/standard/50707.html
 */
```

None: I didn't check @ruudboon comment about `en-US`